tests/cpp/common/test_hist_util.cc

```diff
@@ -347,5 +347,106 @@ TEST(hist_util, SparseCutsExternalMemory) {
     }
   }
 }
+
+TEST(hist_util, IndexBinBound) {
+  uint64_t bin_sizes[] = {static_cast<uint64_t>(std::numeric_limits<uint8_t>::max()) + 1,
```

Comment:
In one case you write: 
```cpp
size_t max_num_bins[] = {256, 65536, 65537};
```
in another case you write:
```cpp
uint64_t bin_sizes[] = {static_cast<uint64_t>(std::numeric_limits<uint8_t>::max()) + 1,
                          static_cast<uint64_t>(std::numeric_limits<uint16_t>::max()) + 1,
                          static_cast<uint64_t>(std::numeric_limits<uint16_t>::max()) + 2};
```

Please, make it consistent.

Reply:
done